core/src/main/java/org/apache/spark/util/collection/TimSort.java

```diff
@@ -120,8 +120,9 @@ public void sort(Buffer a, int lo, int hi, Comparator<? super K> c) {
     assert c != null;
 
     int nRemaining  = hi - lo;
-    if (nRemaining < 2)
+    if (nRemaining < 2) {
```

Comment:
I think we didn't change this file as it's copied as-is from another source. Is there a way to just disable style checks? we could change the file as I doubt we'll ever need to port updates, but who knows

Reply:
> I think we didn't change this file as it's copied as-is from another source. Is there a way to just disable style checks? we could change the file as I doubt we'll ever need to port updates, but who knows

The build errors are all from the java file, the file: scalastyle-config.xml should not work. Do you know which file about java checkstyle works?

Reply:
Look at dev/checkstyle.xml , but you probably just want to suppress them in dev/checkstyle-suppressions.xml